src/transformers/models/mamba2/convert_mamba2_ssm_checkpoint_to_pytorch.py

```diff
@@ -50,6 +56,7 @@ def convert_ssm_config_to_hf_config(config_ssm: Dict, mamba2_model_dict: Dict) -
 
     # Set important values from config and recalculate other resulting entries
     hf_config.hidden_size = config_ssm[config_dict["hidden_size"]]
+    hf_config.intermediate_size = config_ssm[config_dict["intermediate_size"]]
```

Comment:
need to document intermediate size here. The `mixer` has the FF built inside, but this one refers to installing an additional MLP after the mixer / attn blocks

Reply:
not doing. abandoning `modeling_mamba2`